examples/nodejs-alpha/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 Google LLC All Rights Reserved.
```

Comment:
I would suggest moving all of this into the Node.js example -- it's where it will end up eventually.

Having seperate examples for alpha/beta workloads is just going to compound the extra amount of work we need to do as we traverse one level to another.

If we need to hide these features behind a flag passed to the example, I think that's totally fine as well.